site_flow/settings.py

```diff
@@ -54,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': ['templates/home'],
```

Comment:
this should be the absolute path to your root template directory. Settings.py already contains a variable holding the path of the project base directory itself, BASE_DIR, so you can use os.path.join(BASE_DIR, 'templates') in DIRS. If you'd like to see what the result is, simply temporarily add a print(os.path.join(BASE_DIR, 'templates')) at the end of settings.py